src/DynamoCoreWpf/UI/Themes/Modern/OutPorts.xaml

```diff
@@ -101,7 +101,7 @@
                        Grid.Column="1"
                        Height="27px"
                        VerticalAlignment="Center"
-                       Fill="{Binding PortValueMarkerColor, UpdateSourceTrigger=PropertyChanged}"
+                       Fill="#999999"
```

Comment:
Is there a predefined color as this?

Reply:
Nop, the closest one is DarkGrey. https://docs.microsoft.com/en-us/dotnet/api/system.windows.media.colors?view=windowsdesktop-6.0